docs/modules/ROOT/pages/advanced-how-tos/building-olm.adoc

```diff
@@ -0,0 +1,41 @@
+= Building an Operator Lifecycle Manager (OLM) Operator in {ProductName}
+
+If you are developing an application that aligns with the link:https://operatorframework.io/[Operator Framework], and managing it with link:https://olm.operatorframework.io/docs/[Operator Lifecycle Manager (OLM)], you can build that application in {ProductName}. We refer to such applications as OLM Operators. 
+
+OLM Operators include the following elements:
+
+* Operator, which manages Operands
+* Operands, which are custom resources
```

Comment:
components or resources that the operator manages. Better than just saying "custom resources"